### ISSUE TITLE
libgconf2-4 dependency on Debian 10

### ISSUE THREAD

**Issue by ShockwaveNN** [ONLYOFFICE/DocumentServer#528]
Debian 10 (and probably Ubuntu in future releases) remove package `libconf2-4` and renamed it to `libgconf-2-4` (seems some time ago, since both packages are available in current stable releases, but old one not available in Debian 10)

See https://packages.debian.org/stretch/libgconf2-4 and https://packages.debian.org/stretch/libgconf-2-4

Failed to install with error
```
The following packages have unmet dependencies:
 onlyoffice-documentserver : Depends: libgconf2-4 but it is not installable
E: Unable to correct problems, you have held broken packages.
```

**Operating System:** Debian 10 (sid)

**Comment by ShockwaveNN** [comment]
Fixed in https://github.com/ONLYOFFICE/document-server-package/commit/5a99cfa3cdf6d7d48bef879219583a93b3729a5c
Will be released as 5.3.0

**Comment by ShockwaveNN** [comment]
DocumentServer 5.3.0 is released. This issue has been fixed.
